.github/workflows/ci.yml

```diff
@@ -7,7 +7,8 @@ on:
       - '!master'
 
 env:
-  BUILDER_VERSION: v0.6.0
+  BUILDER_VERSION: fixes
+  BUILDER_SOURCE: channels
```

Comment:
This looks like a version used for development in the branch and not meant to be merged into master.

Reply:
Correct. Once that build is released, I'll update this, before pushing